contrib/templates/scala-job/databricks_template_schema.json

```diff
@@ -0,0 +1,57 @@
+{
+  "properties": {
+    "project_name": {
+      "type": "string",
+      "default": "project_name",
+      "description": "Name of project (press enter to use default name)",
+      "order": 1
+    },
+    "jar_dest_path": {
+      "type": "string",
+      "description": "Destination path in Databricks where the JAR will be stored. Note: You must create a Volumes first if you plan to store the JAR there.",
```

Comment:
Maybe better to call this "artifacts_dest_path" to make this more than just JARs.

Add an example to the description